src/main/java/ch/njol/skript/expressions/ExprBlocks.java

```diff
@@ -47,18 +49,22 @@
  * @author Peter Güttinger
  */
 @Name("Blocks")
-@Description("Blocks relative to other blocks or between other blocks. Can be used to get blocks relative to other blocks or for looping.")
+@Description("Blocks relative to other blocks or between other blocks. Can be used to get blocks relative to other blocks or for looping. " +

```

Comment:
maybe we can document that it's a horrible idea to loop all the blocks in a chunk

Reply:
maybe worth noting it includes air as well since most don't consider air to be a block

Reply:
It might be better to add `[(1¦without air)]` so people can choose either?